File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int i=0, j=0, len1=word1.length(), len2=word2.length();
        string ans="";
        while(i<len1 || j<len2){
            if(i<len1)
                ans.push_back(word1[i++]);
            if(j<len2)
                ans.push_back(word2[j++]);
        }
        return ans;
        
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        b = max(len(word1),len(word2))
        a = ''
        for i in range(b):
            if i<len(word1):
                a += word1[i]
            if i<len(word2):
                a += word2[i]
        return a